UI/index.html

```diff
@@ -0,0 +1,535 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="header__wrapper">
+                <a href="#" class="header__wrapper_logo">PROFIT!</a>
+                <button class="header__wrapper_btn">LOG IN</button>
+            </div>
+        </div>
+    </header>
+    
+    <section class="main">
+        <div class="container">
+            <h1>PRODUCTS</h1>
+            <div class="main__filter">
+                <div class="main__filter_by">filter by</div>
+                <div class="main__filter_wrapper">
+                    <button class="main__filter_item">DATE</button>
+                    <div class="main__filter_line">|</div>
+                    <button class="main__filter_item">hashtags</button>
+                    <div class="main__filter_line">|</div>
+                    <button class="main__filter_item">Supplier name</button>
+                </div>
+                <button class="main__filter_reset">reset</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="products">
+       <div class="container">
+           <div class="products__wrapper">
+            <div class="products__item">
+                <div class="products__item_wrapper">
+                    <div class="products__item_leftBlock">
+                        <img class="products__item_icon" src="img/product_item.png" alt="">
+                        <div class="products__item_company">ADOBE</div>
+                        <img src="img/stars.png" alt="" class="products__item_rating">
+                        <div class="products__item_sale">12%</div>
+                        <div class="products__item_date">Until 12.12.2077</div>
+                        <button class="products__item_btnReview">Add ReView</button>
+                        <button class="products__item_btnEdit">EDIT</button>
+                        <button class="products__item_btnDelete">DELETE</button>
+                    </div>
+                    <div class="products__item_rightBlock">

```

Comment:
HTML and CSS are case-insensitive, so please, don't use camel-case classes in HTML here and overall in whole project (use `products-item-right-block` here for example).